### PR TITLE
Feature/vfd swmr

### PR DESCRIPTION
**Pull request by kyang2014**
Add VFD group performance code. The group performance program will be generated, but the program won't be run by the automatic test script.

### CODE REVIEW

File: test/vfd_swmr_gperf_writer.c [hunk 1]
```diff
@@ -0,0 +1,2979 @@
...
+        HDfree(astr_val);
+
+    return false;
+}
+
+/*-------------------------------------------------------------------------
+ * Function:    add_modify_vlstr_attr
+ *
+ * Purpose:     Add a variable length string attribute
+ *              then modify this attribute in this a group.
```
Comment (vchoi-hdfgroup) [inline test/vfd_swmr_gperf_writer.c]: "then modify this attribute in the group."?

File: test/vfd_swmr_gperf_writer.c [hunk 2]
```diff
@@ -0,0 +1,2979 @@
...
+    return false;
+}
+
+/*-------------------------------------------------------------------------
+ * Function:    add_attr
+ *
+ * Purpose:     Add attributes to a group.
+ *
+ * Parameters:  state_t *s
+ *              The struct that stores information of HDF5 file, named pipe
```
Comment (vchoi-hdfgroup) [inline test/vfd_swmr_gperf_writer.c]: Remove "named pipe".  There are other places in the file that mentioned "named pipe".

File: test/vfd_swmr_gperf_writer.c [hunk 3]
```diff
@@ -0,0 +1,2979 @@
...
+
+            if (s.old_style_grp)
+                fprintf(stdout, " Groups: Created via the earlist file format(old-style) \n");
+            else
+                fprintf(stdout, " Groups: Created via the latest file format(new-style) \n");
+
+            fprintf(stdout, "\n");
+
+            fprintf(stdout, "The length of a tick              = %u\n", s.tick_len);
+            fprintf(stdout, "The maximum expected log(in ticks)= %u\n", s.max_lag);
```
Comment (vchoi-hdfgroup) [inline test/vfd_swmr_gperf_writer.c]: Should the sentence be: "The maximum expected lag (in ticks)...."?

**Review by vchoi-hdfgroup — APPROVED** [review]

**Comment by vchoi-hdfgroup** [comment]
Suggest to mention about the performance test somewhere and document the instruction on how to run the test.

File: test/vfd_swmr_gperf_writer.c (hunk 1 above)
Comment (kyang2014) [reply test/vfd_swmr_gperf_writer.c]: Fixed.

File: test/vfd_swmr_gperf_writer.c (hunk 2 above)
Comment (kyang2014) [reply test/vfd_swmr_gperf_writer.c]: Fixed

File: test/vfd_swmr_gperf_writer.c (hunk 3 above)
Comment (kyang2014) [reply test/vfd_swmr_gperf_writer.c]: Fixed

**Pull request merged**